core/integration/steps/table_wrapper.go

```diff
@@ -240,6 +241,13 @@ func (r RowWrapper) MaybeUint(name string) *num.Uint {
 	return u
 }
 
+func (r RowWrapper) MaybeU64(name string) *uint64 {
```

Comment:
Don't like this very much... At the very least, I´d want this renamed to `U64Ptr` or something.

Reply:
What about the consistency with the rest of the functions alike ? Probably another PR, so we don't get side-tracked.